enterprise/server/remote_execution/cgroup/cgroup.go

```diff
@@ -88,8 +88,12 @@ func Setup(ctx context.Context, path string, s *scpb.CgroupSettings, blockDevice
 	for name, value := range m {
 		controller, _, _ := strings.Cut(name, ".")
 		if !enabledControllers[controller] {
-			log.CtxWarningf(ctx, "Skipping cgroup %q setting for disabled cgroup controller %q", name, controller)
-			continue
+			// Attempt to enable the controller if it's not already enabled.
+			if err := EnableController(path, controller); err != nil {
+				log.CtxWarningf(ctx, "Failed to enable cgroup controller %q for cgroup %q: %s", controller, path, err)
+				continue
```

Comment:
Should we keep trying to enable it or memorize that this failed and don't retry?

Reply:
I don't expect this to happen during normal operation, so I don't mind this being noisy for now if it does happen.